content/en/blog/_posts/clientcmd-tutorial.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: blog
+title: 'Uniform API server access using clientcmd'
+date: 2024-07-24
+slug: clientcmd-tutorial
+author: Stephen Kitt (Red Hat)
+---
+
+If you've ever wanted to develop a command-line client for a Kubernetes API,
+especially if you’ve considered making your client usable as a `kubectl` plugin,
+you might have wondered how to make your client feel familiar to users of `kubectl`.
+A quick glance at the output of `kubectl options` might put a damper on that:
+“am I really supposed to implement all those options?”
```

Comment:
“am I really supposed to implement all those options?”
“am I really supposed to implement all those options?”